### DISCUSSION TITLE
Questions for a first-time configuration

### DISCUSSION THREAD

**Discussion by Rom123Soleil** [awawa-dev/HyperHDR#937] [Q&A]
Hello everyone, 

Firstly, thank you @awawa-dev for all the work you've done, both on the forum and on development.   Like many, I've gone through a lot of the internet tutorials, both on Hyperion.Ng and on HyperHDR. It's sometimes difficult to connect the dots as you can find everything and its opposite on the forums. I'm taking the liberty of writing on the forum today because a few questions are still on my mind today.

For context, here's my current configuration:  
* LG OLED B3 (4k120hz and DV compatible)
* Denon AVR3800H (connected via HDMI 2.1 to my TV)
* Xbox One for Blu-Rays and an Android box for Netflix and streaming.

Suggested configuration: 

```txt
Device (Xbox, Android box) => Denon AVR => TV (output 1) 
                                        => Ezcoo splitter (output 2) => MS2130 => Raspberrypi 4
```

desired : 
* Have ambilight on SDR and HDR content.
* Having the option of connecting the PC to the TV via HDMI and retaining 4K120 would be a real plus.

From what I've seen here and there, SDR to HDR switching is easy to do on the HyperHDR web interface, or even potentially automatable with the AVR api => Problem for later.

On the other point, this is where I have several questions: 

* For the Ezcoo splitter, is using HDMI 2.1 interesting? I mean, is it possible for the MS2130 to capture 1080p120? Because obviously the AVR sends the same output on the 2 HDMIs at the same time (In this case, the Ezcoo fakes the EDID and returns the same configuration as the TV, so if the MS2130 doesn't take 120hz, there's no point in using an HDMI 2.1 splitter).
* Will the proposed setup work with Dolby Vision? I've seen a lot of people complaining about a pink screen. Apparently it's not possible to downscale the DV, so do you have to enable and disable the splitter's scaler each time you play Dolby Vision content?

Thanks for your help!

**Comment by awawa-dev** [comment]
Hi
Regarding Denon in different configuration (AppleTV, PS5, dont know Denon model) one of our user experienced strange EDID conflict https://github.com/awawa-dev/HyperHDR/discussions/930 It works till switch HDMI input, it seems if powered on Denon cant provide real HDMI pass-though and changed EDID causing problems. On the other hand Feintech splitter completely failed.
ms2130 can support 1080p120 on input but for low bitrate and DV in LLDV mode: you can try our experimental firmware to get best EDID compatibility.
For LG you can consider piccap so you will have DV and other HDMI2.1 formats. For some input formats the results are excellent but the work is still in progress without any guarantee for your TV model and it's not supported by the mainstream webos grabber... you can read more here: https://github.com/awawa-dev/HyperHDR/pull/896
